transforms:
  T_robot_chess:
    - [-1.0, 0.0, 0.0, 0.358]
    - [0.0, 1.0, 0.0, 0.03]
    - [0.0, 0.0, -1.0, 0.006]
    - [0.0, 0.0, 0.0, 1.0]

# camera 33137761
  T_chess_cam2:
    - [0.9967, 0.0412, -0.0699, 0.0582]
    - [-0.0807, 0.5994, -0.7964, 0.5906]
    - [0.0091, 0.7994, 0.6007, -0.5129]
    - [0.0, 0.0, 0.0, 1.0]

# camera 36829049
  T_chess_cam1:
    - [-0.9947, -0.0872, 0.0539, 0.0757]
    - [0.1018, -0.7737, 0.6253, -0.6511]
    - [-0.0128, 0.6275, 0.7785, -0.5198]
    - [0.0, 0.0, 0.0, 1.0]
